source/index.rst

```diff
@@ -11,3 +16,11 @@ Learn all about COCO here. We introduce you to the concepts and architecture, al
    manage_the_buzz_for_developers
    getting_started
    coco_grove
+
+
+Indices and tables
```

Comment:
Instead of replacing my index.rst with this, please add these lines at the bottom of my index.rst

Reply:
Assuming resolved as previews in browser are fine